webextensions/api/userScripts.json

```diff
@@ -2,58 +2,189 @@
   "webextensions": {
     "api": {
       "userScripts": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/userScripts",
+          "support": {
+            "chrome": {
+              "version_added": "115",
+              "notes": "Available for use with Manifest V3 only."
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "136",
+              "notes": "Available for use with Manifest V3 only."
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "opera": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror"
+          }
+        },
+        "ExecutionWorld": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "115"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "136"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
         "RegisteredUserScript": {
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/RegisteredUserScript",
             "support": {
               "chrome": {
+                "version_added": "115"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "136"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          }
```

Comment:
Should also specify the `worldId` property separately in `RegisteredUserScript` (and `WorldProperties`, below), because Chrome introduced support in 133 (not 115 nor 120).

This is connected to the "multiple user script worlds feature":
- Commit: https://chromium.googlesource.com/chromium/src/+/78e7de28d840e7455b456bf7a7c2c3e6c25e71e4
- Version: https://chromium.googlesource.com/chromium/src/+/78e7de28d840e7455b456bf7a7c2c3e6c25e71e4/chrome/VERSION
- Specification: https://github.com/w3c/webextensions/blob/main/proposals/multiple_user_script_worlds.md